src/meshdb/templates/admin/base.html

```diff
@@ -113,9 +113,6 @@
             {% block footer %}<div id="footer"></div>{% endblock %}
           </div>
         </div>
-        {% block map_sidebar %}
-            {% include "admin/map_sidebar.html" %}
-        {% endblock %}
```

Comment:
Should we remove `map-wrapper` also? 

It's also worth diffing this file with the one it was copied from to see if we can just get rid of it altogether and take the upstream template

Reply:
Diffed it. We include `admin-ext.css` and add this `flex` tag now.

```
░▒~/Code/nycmeshnet/meshdb/.venv▒░ [±wdn/map-iframe-2✘]
$ diff ./lib/python3.13/site-packages/django/contrib/admin/templates/admin/base.html ~/Code/nycmeshnet/meshdb/src/meshdb/templates/admin/base.html
15a16
> <link rel="stylesheet" href="{% static "admin/admin_ext.css" %}">
84c85
<     <div class="main" id="main">
---
>     <div class="main flex" id="main">
113a115
>
```

This also fixes the layout issues you called out in slack